carto_renderer/version.py

```diff
@@ -2,7 +2,7 @@
 A file just to hold the version number, allows automated version increasing.
 """
 
-SEMANTIC = '0.1.6-SNAPSHOT'
+SEMANTIC = '0.1.7-SNAPSHOT'
```

Comment:
There is already a tag for 0.1.6, so this update was missed during the last verison bump.